graphing_outputs/edge_type_comparison_graph.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np 
+import matplotlib.pyplot as plt 
+import pandas as pd
+  
+def edge_count(df_fig1, df_fig2, df_fig3):
+    
+    # define axis
+    cartton1 = pd.read_csv(df_fig1)
```

Comment:
does it matter that this is cartton not cartoon?

Reply:
Good point
Nope, I think we can name it "figure" to be more general

Reply:
I have made all the edits, but I just cannot make the merge for my pull for some reason!